fix (button group): flex wrap with full width not working in frontend

bfintal opened the pull request:
fixes #3135

## Notable fixes in this PR

- Full width buttons and flex wrapping now work correctly
- Button text now doesn't wrap - they look more like what you get in the editor
- Button group vertical alignment now works correctly on multiple buttons (to try it out, hold shift then press return to add a next line)

Commit:
fix (button group): flex wrap with full width not working in frontend

fixes #3135

andeng1106 commented:
@bfintal Scenario 1 is now fixed. One thing I need to confirm for Scenario 2:
- Scenario 2 still looks different compared from editor vs. frontend. Is this okay?

https://github.com/gambitph/Stackable/assets/103395655/f07ce552-aa10-4025-9c33-34c259396ffb

Commit:
fixed fullwidth and changed how buttons collapse.

Commit:
button text no longer wrap

Commit:
button group vertical alignment now works

bfintal commented:
@andeng1106 please note that this PR addresses all these issues:

- Full width buttons and flex wrapping now work correctly
- Button text now doesn't wrap - they look more like what you get in the editor
- Button group vertical alignment now works correctly on multiple buttons (to try it out, hold shift then press return to add a next line)
